includes/phing/build/help/drush.xml

```diff
@@ -144,9 +144,15 @@
             <finally>
                 <trycatch>
                     <try>
+                        <drush command="sqlq" root="${build.platform.dir}" alias="@${drush.alias.default}" bin="${toolkit.dir.bin.drush}" verbose="${drush.verbose}" color="${drush.color}">
```

Comment:
More information about this.

Reply:
Allows to disable and enable modules. Proves to be helpful with platform upgrades that ar missing classes or tables:

* https://drone.fpfis.eu/ec-europa/srb-reference/83/11
```
[error] exception 'PDOException' with message 'SQLSTATE[42S02]: Base table or view not found: 1146 Table 'toolkit.cache_nexteuropa_varnish' doesn't exist' in /test/toolkit/build/includes/database/database.inc:2227
```

* https://drone.fpfis.eu/ec-europa/reps-deutschland-reference/90/11
```
PHP Fatal error: require_once(): Failed opening required '/test/toolkit/build/sites/all/modules/custom/multisite_drupal_language_negociation/multisite_drupal_language_negociation.inc' (include_path='/test/toolkit/vendor/ec-europa/toolkit/vendor/phing/phing/classes:/test/toolkit/vendor/ec-europa/toolkit/vendor/pear/pear_exception:/test/toolkit/vendor/ec-europa/toolkit/vendor/pear/console_getopt:/test/toolkit/vendor/ec-europa/toolkit/vendor/pear/pear-core-minimal/src:/test/toolkit/vendor/ec-europa/toolkit/vendor/pear/archive_tar:/test/toolkit/vendor/ec-europa/toolkit/vendor/pear/versioncontrol_git:.:/usr/share/pear:/usr/share/php') in /test/toolkit/build/includes/language.inc on line 443
```

This will allow us to "temporarily" ask developers to make use of the `drush.rr.disable` property until we have a way to make the deployment steps configurable.